source/draw/RasterYuvSemiplanar.ooc

```diff
@@ -50,7 +50,9 @@ RasterYuvSemiplanar: abstract class extends RasterPlanar {
 		this uv referenceCount decrease()
 		super()
 	}
-
+	apply: override func ~rgb (action: Func (ColorRgb)) {
+		this apply(ColorConvert fromYuv(action))
```

Comment:
I'm not sure if all these `apply` should be done with freeing the Closure. I guess we'll see in tests.